Makefile

```diff
@@ -230,6 +230,12 @@ kubernetes:
 	bash -f .ci/install_bats.sh
 	bash -f integration/kubernetes/run_kubernetes_tests.sh
 
+kubernetes-e2e:
+	cd "${GOPATH}/src/github.com/kata-containers/tests/integration/kubernetes/e2e_conformance" &&\
```

Comment:
does this needs to be full path? or can it be: `cd integration/kubernetes/e2e_conformance`